neo4j/_async/work/session.py

```diff
@@ -321,7 +323,7 @@ async def last_bookmark(self) -> t.Optional[str]:
         if self._auto_result:
             await self._auto_result.consume()
 
-        if self._transaction and self._transaction._closed:
+        if self._transaction and self._transaction._closed():
```

Comment:
this seems unrelated to mypy

Reply:
the linter may point out `closed` is a function, not a property.